torch/csrc/jit/python/init.cpp

```diff
@@ -828,6 +828,10 @@ void initJITBindings(PyObject* module) {
       .def(
           "wait",
           &PythonFutureWrapper::wait,
+          py::call_guard<py::gil_scoped_release>())
+      .def(
+          "_then",
+          &PythonFutureWrapper::then,
```

Comment:
Is RPC and IValue future already merged? :)

Reply:
Yes, they are. See #35154